Sources/DNSMessage/RData/ARecord.swift

```diff
@@ -1,38 +1,92 @@
 import NIO
+public import NIOCore
 
-/// An IPv4 address record. This is used for resolving hostnames to IP addresses.
-public struct ARecord: DNSResource {
-    /// The address of the record. This is a 32-bit integer.
-    public let address: UInt32
+/// [RFC 1035, DOMAIN NAMES - IMPLEMENTATION AND SPECIFICATION, November 1987](https://tools.ietf.org/html/rfc1035)
+///
+/// 3.4.1. A RDATA format
+/// ```
+///     +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+///     |                    ADDRESS                    |
+///     +--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+--+
+/// ```
+/// where:
+///
+/// ADDRESS         A 32 bit Internet address.
+public struct ARecord: DNSResourceData {
+    public let ipv4Address: [UInt8]
```

Comment:
```suggestion
    public let bytes: [UInt8]
```

This would be more appropriate. We can also expose it as `InlineArray<4, UInt8>` to save a malloc

Reply:
Wouldn't we have to bump the version to 6.2 to use InlineArrays?